.env.test

```diff
@@ -0,0 +1,14 @@
+MAFIASI_DEBUG=True
+MAFIASI_ENABLE_JABBER_INTEGRATION=False
+MAFIASI_ENABLE_LDAP_AUTH_BACKEND=False
+MAFIASI_ENABLE_LDAP_REGISTRATION=False
+MAFIASI_ENABLE_MUMBLE_INTEGRATION=False
+MAFIASI_ENABLE_EP_INTEGRATION=False
+MAFIASI_SECRET_KEY=django-insecure-foobar123
+MAFIASI_EMAIL_HOST=mailhost.informatik.uni-hamburg.de
+MAFIASI_EMAIL_ADDRESSES_PASSWORD=
+MAFIASI_DB=postgres://mafiasi-dashboard:mafiasi-dashboard@localhost/mafiasi-dashboard
```

Comment:
Is it really a good idea to use the same database for tests? Is the test data removed after the tests?

Reply:
Also, I would vastly prefer to change the database name and user to mafiasi_dashboard (underscore instead of minus) to go with common sql naming conventions.

Reply:
> Is the test data removed after the tests?

Yes it is. It might be that django recreates the database from scratch before tests run too; I would need to look into that

Reply:
I looked it up and yes, django creates a whole new test database for tests that is automatically created and purged.
See the [docs](https://docs.djangoproject.com/en/4.1/topics/testing/overview/#the-test-database) for more info.